hardware_defines.h

```diff
@@ -46,6 +46,10 @@
 #define CACHED_FUNCTION_ATTR
 #endif
 
+#if IS_RP2040()
+#define AUDIO_RATE_PLATFORM_DEFAULT 32768
+#endif
+
```

Comment:
I do not think this is needed as the RP2040 is not AVR so the test on line 35 will be false and the AUDIO_RATE was already at 32768.

Reply:
Have you checked this?